web/src/layouts/UsersAdministrationLayout/UsersAdministrationLayout.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC, ReactNode } from "react";
+import { useLocation } from "react-router-dom";
+
+import { Box, Theme, Button as MuiButton } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+import { Link } from "@components/Link/Link";
+import { ArrowLeftIcon } from "@components/icons/ArrowLeftIcon/ArrowLeftIcon";
+import { PageLayout } from "@layouts/PageLayout/PageLayout";
+import { TopAppBar } from "@layouts/TopAppBar/TopAppBar";
+import { rgbaColors } from "@themes/palette";
+import { Paths } from "@routing/paths";
+
+interface Props {
+  onBackClick?: () => void;
+  buttons: ReactNode;
+}
+
+export const UsersAdministrationLayout: FC<Props> = ({
+  children,
+  onBackClick,
+  buttons,
+}) => {
+  const theme = useTheme<Theme>();
+  const location = useLocation();
+
+  const isUsersTab = location.pathname.split("/")[2] === "users";
+
+  const getTabStyles = (isTabActive: boolean) => ({
+    color: isTabActive
+      ? theme.palette.primary.main
+      : theme.palette.common.black,
+    opacity: isTabActive ? 1 : 0.6,
+    fontWeight: 600,
+    lineHeight: "16px",
+    height: "100%",
+    borderBottom: isTabActive
+      ? `1px solid ${theme.palette.primary.main}`
+      : `1px solid ${theme.palette.common.white}`,
+    borderRadius: 0,
+  });
+
+  return (
+    <>
+      <TopAppBar showLogo={false} />
+      <PageLayout
+        py={5}
+        px={2}
+        mt={0}
```

Comment:
Is there a need to declare `mt={0}`. Shouldn't `PageLayout` have `0` ootb, so parent components are able to align it without problems?

Reply:
I had to because <PageLayout /> has margin-top set to 32px by default. And I didn't wanted it in my case.